readme.txt

```diff
@@ -34,6 +34,43 @@ make
 then as root
 make install
 
+
+Customize Desktop Environment for xRDP Session
```

Comment:
Please use "XRDP" (preferred for the whole software package) or "xrdp" (preferred for the executable file), but not any other capitalization.